doc/03_reference/cs_registers.rst

```diff
@@ -576,6 +582,38 @@ The User Mode ``time(h)`` registers are not implemented in Ibex.
 Any access to these registers will trap.
 It is recommended that trap handler software provides a means of accessing platform-defined ``mtime(h)`` timers where available.
 
+Machine Vendor ID (mvendorid)
+-----------------------------
+
+CSR Address: ``0xF11``
+
+Reset Value: ``0x0000_0000``
+
+Use the ``CSR_MVENDORID_VALUE`` parameter in :file:`rtl/ibex_pkg.sv` to change the fixed value.
+Details of what the ID represents can be found in the RISC-V Privileged Specification, version 1.11.
```

Comment:
```suggestion
Details of what the ID represents can be found in the RISC-V Privileged Specification.
```

(We specify the version of the RISC-V spec that we follow in one place to avoid having to update them everywhere if we change it.)

Also below.